app/services/quote_service.rb

```diff
@@ -0,0 +1,11 @@
+class QuoteService
+
```

Comment:
This looks good. You could add the keyword `private` and move your `self.conn` method below that. 
```
self.search_by_keyword(keyword) method

private

self.conn method
```